lib/push.js

```diff
@@ -6,13 +6,32 @@ var request = require('request')
 
 var ERR_NO_PUSH_CALLBACK = 'No Push Callback'
 
-var LOG_UPDATE_INCREMENT_SEND = 'push.send'
-var LOG_UPDATE_INCREMENT_SUCCESS = 'push.success'
-var LOG_UPDATE_INCREMENT_RESET_SETTINGS = 'push.reset_settings'
-var LOG_UPDATE_INCREMENT_FAIL = 'push.failed'
-var LOG_UPDATE_INCREMENT_NO_CALLBACK = 'push.no_push_callback'
 var LOG_OP_NOTIFY_UPDATE = 'push.notifyUpdate'
 
+var reasonToEvents = {
+  accountVerify: {
```

Comment:
I kept the original verification reasons, since I assume we have existing data for them. @vladikoff, please let me know if you'd like me to qualify these, too.